### DISCUSSION TITLE
Team up! (Find potential teammates here) 👩🏽‍🤝‍👨🏻

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#13] [General]
Some participants have reached out and expressed that they'd like to find team members. This is a good opportunity to combine skills! If you're looking for a team member, please reply with:

- What skills you're looking for (if any)
- Your timezone/availability to meet
- Preferred form of communication (email, discord, etc)
- Some way to contact you

**Comment by hadil1999-creator** [comment]
Hi, Looking for teammates, 
Availability any day - time zone GMT-5
Linkedin to set a first contact: https://www.linkedin.com/in/hadil-ben-amor-777a981ba/

**Reply by yusciti** [reply]
Hello everyone
Am seriously looking for group of teammates.
Kindly reach out to me on https://www.linkedIn.com/in/yusuph-taopheeq-baab2280

**Reply by osama-swe** [reply]
Hello @hadil1999-creator and @yusciti I'm from Cairo, Egypt (GMT+3)
Let's Connect: https://www.linkedin.com/in/osama-ayman/

**Reply by minahil-105** [reply]
I am also looking for team LangchainJS

**Reply by Sirius-ashwak** [reply]
We can form a team together.
https://www.linkedin.com/in/mohamed-ashwak-m-b8835a262/

**Comment by nayanbanik** [comment]
Hey folks. Hope you are doing well. I am looking for teammates with expertise in Python and RAG architecture. If you are interested or know someone who has the potential, connect here: https://www.linkedin.com/in/iamnayan/

**Reply by musaqlain** [reply]
@nayanbanik let's connect:

I am a software engineer with extensive experience in full-stack development. I have worked with technologies like Next.js, React, JavaScript, and TypeScript, and I am skilled in state management using Redux and Redux Toolkit.

On the back end, I have experience with Node.js, Express.js, and REST APIs. I am proficient in databases such as MongoDB, MySQL, and PostgreSQL. Additionally, I am familiar with cloud and DevOps tools like AWS (RDS, S3, EC2, Lambda), Docker, CI/CD, and Kubernetes.

I have built projects like ShopO and eCommerc3, integrating technologies like Stripe, PayPal, and OAuth API, and using modern UI libraries such as Material UI and Tailwind CSS (also mentioned in my resume). I have also developed AI-powered applications like QuillPDF using LangChain, Pinecone, Prisma, and Next.js.

LinkedIn: https://www.linkedin.com/in/muhammad-saqlainraza/
GitHub: https://github.com/MuhammadSaqlain-dev
🙌

**Reply by temesgen5335** [reply]
hey modammad I would love to team up, lets connect. linkedin.com/in/temesgen-gebreabzgi

**Reply by Bipan101** [reply]
Yeah take me too.

**Comment by minahil-105** [comment]
Heyyy, Team for LangchainJS 
If you're interested comment below

**Comment by Mr-Nnobody** [comment]
Hey , looking for teammates. 
Preferred language : python
Due to time zone troubles we'd prefer someone based Africa. (We welcome all levels expertise)
If interested , reach out by LinkedIn 
https://www.linkedin.com/in/lewis-tem-bueh

**Comment by faizan-yousaf** [comment]
I am looking for teammates with expertise RAG architecture and LangChain. If you are interested and have the potential, connect here: https://www.linkedin.com/in/mrfaizanyousaf/

**Comment by xXWarMachineRoXx** [comment]
Hi Folks, Hope you're doing great and are excited for this project. My Team's Name is [**Leadcode.xyz** ](https://discord.gg/YpHmZ3Jg). Join our discord server and lets talk about the amazing things we're gonna hack 🔥


# Let's Connect 

**What skills you're looking for (if any):** Ideation skills and RAG experience
**Your timezone/availability to meet :**    6PM to 10PM (GMT+ 5:30 )
**Preferred form of communication (email, discord, etc) :** active in mails, discord , or WhatsApp. I Prefer discord for event related discussions.
**Some way to contact you:**  But, First you must contact me through my [discord](lucifer_is_back_)

# About me 
**Team Name**: Leadcode.xyz
**Preferred Language** : python > js > c++, c# > ruby
**Work experience** : 3 years in software dev. (1 year in Unified communications platforms , 2 years in AI and Software Development)
**My Batcave is located in** : India. 🇮🇳

**Reply by mrk1989** [reply]
I can't send a message because not have premium linkedin.
we can collaborate and maybe send e-mail to mrk1989@hotmail.it

**Reply by xXWarMachineRoXx** [reply]
> I can't send a message because not have premium linkedin. we can collaborate and maybe send e-mail to [mrk1989@hotmail.it](mailto:mrk1989@hotmail.it)

hmm send me your discord id then

**Reply by temesgen5335** [reply]
hey i want to team up. lets connect:
linkedin.com/in/temesgen-gebreabzgi
discord username: temesgen5335

**Comment by ozdemiremrah81** [comment]
Hello Everyone,
I just want to collaborate on your project with my experience in System Engineering. I can research anything related to the cloud, infrastructure, network or devops. 
I'm available between 3am-9pm UTC.
Linkedin: https://www.linkedin.com/in/ozdemiremrah/
Github: https://github.com/ozdemiremrah81

**Comment by Yash-Kavaiya** [comment]
**Hello Everyone,**

I am looking for teammates to collaborate on this project. I have participated in over 10 AI hackathons and am seeking senior members to guide and help improve our project.

**LinkedIn**: [Yash Kavaiya](https://www.linkedin.com/in/yashkavaiya/)  
**GitHub**: [Yash-Kavaiya](https://github.com/Yash-Kavaiya)

**Comment by TNAHOM** [comment]
Anyone with nextjs or react background. Contact me to join my group
LinkedIn: [https://www.linkedin.com/in/nahom-tamirat-ab7031232/](url)

**Reply by minahil-105** [reply]
Sent u request, let's do it
